### DISCUSSION TITLE
Repeatedly get the following error in Godot with the Dialogue Manager extension:

### DISCUSSION THREAD

**Discussion by audreyleighleigh** [nathanhoad/godot_dialogue_manager#250] [Q&A] [answered]
const _DialogueManager = preload("res://addons/dialogue_manager/dialogue_manager.gd")

  start: Script does not inherit from Node: res://addons/dialogue_manager/dialogue_manager.gd.
  <C++ Error>    Condition "!valid_type" is true. Continuing.
  <C++ Source>   main/main.cpp:2776 @ start()

I've tried uninstalling and reinstalling Dialogue_Manager, and I've also updated the latest version. I'm on a MacBook pro running Montery. Any ideas as to what may be causing this issue?

**Comment by nathanhoad** [comment, marked as answer]
Are you using Godot 4.1? If not, try that.

**Reply by audreyleighleigh** [reply]
That resolved the problem! Thank you so much!
